integration/311_container_to_container_edge_with_ebpf_test.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+
+# shellcheck disable=SC1091
+. ./config.sh
+
+start_suite "Test short lived connections between containers, with ebpf connection tracking enabled"
+
+weave_on "$HOST1" launch
+scope_on "$HOST1" launch --probe.ebpf.connections=true
+weave_on "$HOST1" run -d --name nginx nginx
+weave_on "$HOST1" run -d --name client alpine /bin/sh -c "while true; do \
+	wget http://nginx.weave.local:80/ -O - >/dev/null || true; \
```

Comment:
Same comment as in the example above (why so many connections?). Also, if possible please use curl for consistency with the rest of the tests.